samples/RaspberryPiInstructions.md

```diff
@@ -11,6 +11,7 @@ These steps have been tested on a RPi 2 and RPi 3 with Linux and Windows.
 
 * From the terminal/commandline create a folder named helloworld and go into it.
 * Run `dotnet new console -n helloworld`.
+* Change directory into `helloworld` and you can find helloworld.csproj file.
```

Comment:
If we're going to tell people here to change to the helloworld directory, we should delete the words telling people to do that from two lines higher.